packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+import * as React from 'react';
+
+export declare class FocusManager {
```

Comment:
>FocusManager [](start = 21, length = 12)

Why does this file have "win32" in it? It's already in react-native-win32 folder

Reply:
I was just following the patterns from the package. Every file in this Utilities folder has the ".win32"

Reply:
Platform extensions are related to how metro bundles platform specific files.